### PR TITLE
changelog #12719

### PR DESCRIPTION
**Pull request by ijobling**

### CODE REVIEW

**Comment by ijobling** [comment]
adding ability to select when to show answer/rationale, and moving partial points in advanced code test to grading tab

File: source/instructors/authoring/assessments/advanced-code-test.rst [hunk 1]
```diff
@@ -40,9 +40,9 @@ Complete each section to set up your advanced code test.
...
   - **One Attempt Only** - Toggle to enable the assessment to run only once. The student will be warned that they cannot resubmit the assessment. It's recommended that you provide a :ref:`Run Button <customizable-run-menu>` for the student to test the code before running the actual assessment.
-  - **Show Answer and Rationale to Students** - Toggle to display the answer, and the rationale for the answer, to the student. This guidance information will be shown to students after they have submitted their answer and any time they view the assignment after marking it as completed. 
+  - **Show Answer and Rationale to Students** - Toggle to display the answer, and the rationale for the answer, to the student. This guidance information will be shown to students after they have submitted their answer and any time they view the assignment after marking it as completed. You can set when to show this selecting from **Mever**, **After x attempts**, **If score is greater than or equal to a % of the total** or **Always**
```
Comment (shajason) [inline source/instructors/authoring/assessments/advanced-code-test.rst:45]: Typo - Never

File: source/instructors/authoring/assessments/autograde-free-text.rst [hunk 2]
```diff
@@ -49,7 +49,7 @@ Follow these steps to set up an autograde free text assessment:
     - **Plaintext + LaTeX** - Students enter plaintext with no markdown formatting but offers support for LaTeX commands. A preview window is available where students can see the rendered LaTeX output.
     - **Markdown + LaTeX** - Students enter markdown that also offers support for LaTex commands. A preview window is available where students can see the rendered markdown with LaTeX output.
 
-  - **Show Answer and Rationale to Students** - Toggle to display the answer, and the rationale for the answer, to the student. This guidance information will be shown to students after they have submitted their answer and any time they view the assignment after marking it as completed. 
+  - **Show Answer and Rationale to Students** - Toggle to display the answer, and the rationale for the answer, to the student. This guidance information will be shown to students after they have submitted their answer and any time they view the assignment after marking it as completed. You can set when to show this selecting from **Mever**, **After x attempts**, **If score is greater than or equal to a % of the total** or **Always**
```
Comment (shajason) [inline source/instructors/authoring/assessments/autograde-free-text.rst:52]: Typo Never

File: source/instructors/authoring/assessments/fill-in-blanks.rst [hunk 3]
```diff
@@ -80,7 +80,7 @@ Follow these steps to set up fill in the blank assessments:
 
   - **Show Expected Answer** - Toggle to show the students the expected output when they have submitted an answer for the question. To suppress expected output, disable the setting. 
 
-  - **Show Answer and Rationale to Students** - Toggle to display the answer, and the rationale for the answer, to the student. This guidance information will be shown to students after they have submitted their answer and any time they view the assignment after marking it as completed.
+  - **Show Answer and Rationale to Students** - Toggle to display the answer, and the rationale for the answer, to the student. This guidance information will be shown to students after they have submitted their answer and any time they view the assignment after marking it as completed. You can set when to show this selecting from **Mever**, **After x attempts**, **If score is greater than or equal to a % of the total** or **Always**
```
Comment (shajason) [inline source/instructors/authoring/assessments/fill-in-blanks.rst:83]: Typo Never

**Review by shajason — APPROVED** [review]

**Pull request merged**